### PR TITLE
Updated bffnt.py

### PR DESCRIPTION
**Pull request by Tyulis**
Fixed Python3 compatibility, added support of version 3 (CFNT and CFNU)

### CODE REVIEW

**Comment by Tyulis** [comment]
I cannot test with python2, there is no reasons for new issues to appear, but if you can test, it would be better

File: bffnt.py [hunk 1]
```diff
@@ -503,32 +542,38 @@ def save(self, filename):
...
-        if version != VERSION:
+
+        if self.version not in VERSIONS:
             print('Unknown version: 0x%08x (expected 0x%08x)' % (version, VERSION))
             self.invalid = True
             return
 
         if header_size != FFNT_HEADER_SIZE:
             print('Invalid header size: %d (expected %d)' % (header_size, FFNT_HEADER_SIZE))
+            blinh
```
Comment (ObsidianX) [inline bffnt.py]: Typo?

**Comment by Tyulis** [comment]
First issue: this is because in Python3, the / operator makes always a float, and an integers are required, this make errors. For the second... ... Don't know. But it's not wanted...

**Comment by Tyulis** [comment]
Sorry for the delay. It's now fixed.

**Comment by ObsidianX** [comment]
Thanks for the PR and the Python 3 division lesson!

**Pull request merged**